std_msgs/Header header
struct_slam/plm0[] pclP
